### ISSUE TITLE
lsdyna

### ISSUE THREAD

**Issue by bxsjiab** [Ice-Citron/FEA-Physics_IA#1]
Hello, have you solved the problem of em solver not converging? I had the same problem recently.

**Comment by Ice-Citron** [comment]
Yes. but its cuz my stuff is too small. i recommend scaling up too idk

but, i mainly recommend trying to find a mentor from fiverr. there's plenty of people that teaches u or makes u simulation stuff in Fiverr from Pakistan, India, etc. for like 50-100 USD or maybe 10USD~

**Comment by Ice-Citron** [comment]
@bxsjiab

**Comment by bxsjiab** [comment]
ok，thank you
